config.md

```diff
@@ -25,6 +25,8 @@ For example, if an implementation is compliant with version 1.0.1 of the spec, i
 **`root`** (object, required) configures the container's root filesystem.
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  The path MUST be an a relative path (not starting with /), which is relative to the bundle. e.g.
```

Comment:
Acutally, I think no matter absolute path or relative path is OK.
I just want to clarify which kind of value can be set  to path.